client/src/app/pages/login-signup/login-signup.component.scss

```diff
@@ -0,0 +1,98 @@
+@import "../../../styles.scss";
+
+.container {
+    mix-blend-mode: screen;
```

Comment:
I removed this line and tried the home page and I couldn't see a change, so maybe it is safe to delete this line?

Reply:
I think you're right, I had improper implementation of that css styling